client/tailscale/localclient.go

```diff
@@ -259,6 +259,22 @@ func (lc *LocalClient) DaemonMetrics(ctx context.Context) ([]byte, error) {
 	return lc.get200(ctx, "/localapi/v0/metrics")
 }
 
+// ClientMetric is the request data for UploadClientMetric.
+type ClientMetric struct {
+	Name string `json:"name"`
+	// One of "counter" or "gauge"
```

Comment:
// Type is ...

And say that it's only used if the metric named by Name doesn't already exist?

And document the others.